engine/apps/api/permissions.py

```diff
@@ -310,18 +350,23 @@ def has_object_permission(self, request: AuthenticatedRequest, view: ViewSetOrAP
 
 
 ALL_PERMISSION_NAMES = [perm for perm in dir(RBACPermission.Permissions) if not perm.startswith("_")]
-ALL_PERMISSION_CLASSES = [
+ALL_PERMISSION_CLASSES: LegacyAccessControlCompatiblePermissions = [
     getattr(RBACPermission.Permissions, permission_name) for permission_name in ALL_PERMISSION_NAMES
 ]
-ALL_PERMISSION_CHOICES = [
+ALL_PERMISSION_CHOICES: typing.List[typing.Tuple[str, str]] = [
     (permission_class.value, permission_name)
     for permission_class, permission_name in zip(ALL_PERMISSION_CLASSES, ALL_PERMISSION_NAMES)
 ]
 
 
-def get_permission_from_permission_string(perm: str) -> typing.Optional[LegacyAccessControlCompatiblePermission]:
+def get_permission_from_permission_string(
```

Comment:
(see [here](https://github.com/grafana/oncall/blob/jorlando/update-rbac-permissions-to-support-irm-app/engine/apps/api/views/user.py#L189) for more context on how this is used; _tldr;_ query filtering for the user's endpoint)